devicetypes/smartthings/zwave-switch-generic.src/zwave-switch-generic.groovy

```diff
@@ -179,17 +176,10 @@ def ping() {
 }
 
 def refresh() {
```

Comment:
I'd just have `refresh()` call `poll()` or vice versa. Not clear why it wasn't that way already.

Reply:
Probably poll should call refresh bc we might be removing poll soon since ping fills its role.